style/tailwind.css

```diff
@@ -1,141 +1,264 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-
-.markdown-content {
-  @apply bg-base-200 mt-4 p-5 rounded space-y-4;
-}
-
-.markdown-content h1 {
-  @apply text-2xl font-black;
+@import "tailwindcss" source("../src");
```

Comment:
Adding `source("../src")` afterwards is what fixed the mysterious CI build issue.

Reply:
Nice, that's a weird one.